.gitignore

```diff
@@ -1 +1,2 @@
 lib
+node_modules
```

Comment:
not sure if that's intentional, but if I leave this off, I need to manually delete or ignore the node_modules, so I just included it here